.github/workflows/docker-publish.yml

```diff
@@ -0,0 +1,27 @@
+name: Docker Publishing
+
+on:
+  push:
+    branches:
+      - "*"
+    tags:
+      - "[0-9]+.[0-9]+.[0-9]+"
+
+jobs:
+  publish:
+    runs-on: ubuntu-20.04
```

Comment:
We should be using 22.04. This will go stale fast. Might as well be up to date now.